python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1006,6 +1100,26 @@ def multiply(x1, x2, out=None):
 
 @set_module('mxnet.symbol.numpy')
 def divide(x1, x2, out=None):
+    """Returns a true division of the inputs, element-wise.
+
+    Parameters
+    ----------
+    x1 : _Symbol
+        Dividend array.
+
+    x2 : _Symbol
+        Divisor array.
+
+    out : ndarray
```

Comment:
`out` is also _Symbol

Reply:
I just learned from you that it should be "Dummy parameter to keep the consistency with the ndarray counterpart" :)

Reply:
This is from the template:
```
out : _Symbol or None
        Dummy parameter to keep the consistency with the ndarray counterpart.
```

Reply:
Awesome.